ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.21.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  base-machine:
+    name: fedora
+    namespace: openshift
+    tag: "29"
+  machine-os-content-base:
+    name: "4.8"
+    namespace: ocp
+    tag: machine-os-content
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```

Comment:
Shouldn't we be using a newer golang?

Reply:
Yup, other 4.8 jobs are using 1.14, bumped it up.